docs/io.cozy.contacts.md

```diff
@@ -84,6 +84,24 @@ Used to group contacts together. A group can obviously have multiple contacts, b
 - `trashed`: {boolean} is `true` if the group is marked for removal and will be deleted soon
 - `metadata`: {object}
   - `version`: {integer} Used for migrations. Current version is **1**
+- `relationships`: {object} links between documents
+
+  - `accounts`: {object}
+    - `data`: {array} list of accounts
+      - `_id`: {string} id of the io.cozy.contacts.accounts document
+      - `_type`: {string} doctype "io.cozy.contacts.accounts"
+- `cozyMetaData`: {object}
+
+  - ...`cozyMetadata`: [Document metadata](./README.md#document-metadata)
+  - `sync`: {object}
+
+    - `ded4265b38c54b0683408c76d9ebd`: {object} id of the sourceContactsAccount
```

Comment:
1. "sourceContactsAccount" ? What's that ?
2. Why using a keyed object as root and not an array of objects with this id as attribute?

Reply:
1. It's the `io.cozy.contact.account` that this group is linked to. `io.cozy.contact.account` is a "copy" of `io.cozy.accounts` but without password access and more sync infos.
2. Not sure why we decided that, it's the same than `io.cozy.contacts`. I think it might be for easier indexing, but I can't really remember.

Reply:
Then you probably should at least reference "io.cozy.constact.account" and not "sourceContactsAccount". 

However, if so, is it a duplicate from the `constactsAccountsId` later ? Do you really need this keyed object if it generates a duplication a few lines later?

Reply:
@Crash-- @cedricmessiant what do you think?

I can't remember why we decided to duplicate the information, but it's probably not necessary. However changing it involves a lot of work (both contact connectors + the app) so I don't think it's worth changing.

Reply:
The duplication of the information is not necessary but the use of a keyed object is practical to be able to use `lodash.get` for example and not having to do `.find` to search for the contact account that we need.
I think that changing it to an array of object is a lot of work. Removing the id attribute is probably easier but I'm still not sure it worths it for now.

Reply:
I am not really in favor of removing the attribute. Let's leave that as is if it's a lot of work with no expected earning.